src/main/java/com/redhat/coolstore/model/ShoppingCart.java

```diff
@@ -1,115 +1,111 @@
-package com.redhat.coolstore.model;
+// Update the import statements
+import jakarta.enterprise.context.Dependent;
+import jakarta.enterprise.context.SessionScoped;
 
-import java.io.Serializable;
-import java.util.ArrayList;
-import java.util.List;
-
-import javax.enterprise.context.Dependent;
-
-@Dependent
```

Comment:
Not sure why it removed `@Dependent` here, there is no rule for that in our inventory